content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -103,38 +107,40 @@ Upgrading the Istio CNI agent to a compatible version in-place will not disrupt
 $ helm upgrade istio-cni istio/cni -n istio-system
 {{< /text >}}
 
-### Upgrade the Gateway component (optional)
+## Upgrade waypoints and gateways using tags
 
-Gateway components manage east-west and north-south dataplane traffic between ambient mesh boundaries, as well as some aspects of the L7 dataplane.
+If you have followed best practices, all of your gateways, workloads, and namespaces use either the default revision (effectively, a tag named `default`), or the `istio.io/rev` label with the value set to a tag name. You can now upgrade all of these to the new version of the Istio data plane by moving their tags to point to the new version, one at a time. To list all tags in your cluster, run:
 
-{{< text syntax=bash snip_id=upgrade_gateway >}}
-$ helm upgrade istio-ingress istio/gateway -n istio-ingress
+{{< text syntax=bash snip_id=list_tags >}}
+$ kubectl get mutatingwebhookconfigurations -l 'istio.io/tag' -L istio\.io/tag,istio\.io/rev
```

Comment:
No equivalent istioctl cmd for it? :-(. would `istioctl tag list` work?

Reply:
Since this is a helm-based guide, we chose to reference istioctl in as few locations as possible.

Reply:
We are stuck `getting` a MWH config because `istioctl` does that under the hood - it is A Problem that `istioctl` masks for us, and any other tool will not.